docs/4.1/admin-guide.md

```diff
@@ -2081,16 +2081,10 @@ teleport:
   storage:
       # The region setting sets the default AWS region for all AWS services 
       # Teleport may consume (DynamoDB, S3)
-      region: us-west-1
+      region:  us-east-1 
 
-      # Path to S3 bucket to store the recorded sessions in. The optional 'region'
-      # parameter allows to override the region setting above, keeping S3 recordings
-      # in a different region:
-      audit_sessions_uri: s3://example.com/path/to/bucket?region=us-east-1
-
-      # Authentication settings are optional (see below)
-      access_key: BKZA3H2LOKJ1QJ3YF21A
-      secret_key: Oc20333k293SKwzraT3ah3Rv1G3/97POQb3eGziSZ
+      # Path to S3 bucket to store the recorded sessions in.
+      audit_sessions_uri:  s3://Example_TELEPORT_S3_BUCKET/records
```

Comment:
add quotes, let's use quotes consistently everywhere to avoid potential yaml errors